From 894b49877282bfd5c90ce1c8bd18c174e6e73173 Mon Sep 17 00:00:00 2001
From: Marvin Friede <51965259+marvinfriede@users.noreply.github.com>
Date: Mon, 11 Nov 2024 09:39:33 -0600
Subject: [PATCH] Access parameters on Windows

---
 python/dftd4/parameters.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/python/dftd4/parameters.py b/python/dftd4/parameters.py
index 21676233..98ea1498 100644
--- a/python/dftd4/parameters.py
+++ b/python/dftd4/parameters.py
@@ -53,7 +53,12 @@ def get_data_file_name(base_name: str = "parameters.toml") -> str:
         dirname(__file__), "..", "..", "..", "..", "share", "dftd4", base_name
     )
     if not exists(data_file):
-        data_file = join(dirname(__file__), base_name)
+        # for Windows install layout
+        data_file = join(
+            dirname(__file__), "..", "..", "..", "Library", "share", "s-dftd3", base_name
+        )
+        if not exists(data_file):
+            data_file = join(dirname(__file__), base_name)
 
     return data_file
 
